tasker/tasker_ctl.py

```diff
@@ -78,27 +78,41 @@ def _resolve_static_task(self, taskname, world=None) -> dict:
         stage = self.taskfile_obj.tasks[taskname]
         next_stages = {}
 
+        def add_stage_node_to_jorney(taskname, next_stage):
+            if next_stage not in world['known']:
+                world['unknown'].update({next_stage: taskname})
+                world['jorney'].append(f'{taskname} --> stage("{next_stage}")')
```

Comment:
should've use 'full_jorney' field
keep original field as it was, it will keep compatibilty